resources/idlharness.js

```diff
@@ -1529,12 +1529,13 @@ IdlInterface.prototype.test_self = function()
         // https://github.com/heycam/webidl/issues/698
         assert_true(isConstructor(this.get_interface_object()), "interface object must pass IsConstructor check");
 
+        // "If I was not declared with a constructor operation, then throw a TypeError."
```

Comment:
This comment should probably be removed as it no longer reflects the current spec. And, it would only apply to the part inside the `if`.